fix: BROS-622: When exporting YOLO with images format, the exported file does not contain image

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/20440611134)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/706
<!-- Follow Merge -->

makseq committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20432436217

makseq committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20432699522

makseq committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20435528095

makseq committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20436487746

makseq committed:
fix: BROS-622: When exporting YOLO with uploaded images format from persistent storage, the exported file does not contain image

makseq committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20439894205

makseq committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20439957428

makseq committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20440429880

makseq committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20440490323

makseq committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20440611134

makseq commented:
/fmt all

farioas commented:
/fmt
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20441474554)

makseq commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20450084879)
> Successfully merged:  1 file changed, 2 deletions(-)

makseq committed:
Merge branch 'develop' into 'fb-bros-622'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20450084879

makseq commented:
/fmt
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20450169200)

makseq committed:
Apply pre-commit linters

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20450169200

makseq commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20471840015)
> Successfully merged:  create mode 100644 label_studio/fsm/tests/test_task_annotation_workflows.py

makseq committed:
Merge branch 'develop' into 'fb-bros-622'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20471840015

makseq commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/20488767356)
> Successfully merged:  1 file changed, 27 insertions(+)

makseq committed:
Merge branch 'develop' into 'fb-bros-622'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/20488767356

makseq committed:
Merge branch 'develop' into fb-bros-622
